packages/account/src/Sections/Security/ConnectedApps/connected-apps.jsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useState, useContext, useCallback, useMemo } from 'react';
```

Comment:
We should  not import `React, { useState, ....`

Let's use `React.useContext`, `React.useState`, etc. This is intended approach for performance reasons.

Reply:
For consistency reasons I updated import, But I was wondering isn't this favouring micro-optimization over readability and maintainability?